Photon-Tinker/SPKTinkerViewController.m

```diff
@@ -597,9 +595,9 @@ - (void)pinCallHome:(PinView *)pinView
             [pinView endUpdating];
             
             NSString* errorStr = [NSString stringWithFormat:@"Error communicating with device (%@)",errorMessage];
-            [[Mixpanel sharedInstance] track:@"Tinker: error" properties:@{@"type":@"communicate with device"}];
+            [[SEGAnalytics sharedAnalytics] track:@"Tinker: error" properties:@{@"type":@"communicate with device"}];
```

Comment:
Did we check that analytics events are flowing correctly now?

Reply:
Checking with jberi.

Reply:
Seems to be working in Segment, need you to check in Firebase.